static/js/server_events_dispatch.js

```diff
@@ -189,9 +189,9 @@ export function dispatch_normal_event(event) {
                 allow_edit_history: noop,
                 allow_message_editing: noop,
                 edit_topic_policy: noop,
-                user_group_edit_policy: noop,
                 avatar_changes_disabled: settings_account.update_avatar_change_display,
                 bot_creation_policy: settings_bots.update_bot_permissions_ui,
+                user_group_edit_policy: settings_user_groups.update_user_groups_settings_tip,
```

Comment:
Why this line is moved here from above? We usually try to avoid doing such unnecessary changes in the same commit.